### PR TITLE
Added token auth to `CondaSlice` and auth IT

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #57 
Added token auth to `CondaSlice` and integration test with auth. Note that for install, conda can use as token-based auth (if user was logged in with `anaconda login`) as basic auth (when user was not logged and channel is set with credentials).

### CODE REVIEW

File: src/test/java/com/artipie/conda/CondaSliceAuthITCase.java [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+            new StringContainsInOrder(
+                new ListOf<>(
+                    "http://host.testcontainers.internal",
+                    "Alice/snappy/1.1.3/linux-64/snappy-1.1.3-0.tar.bz2",
+                    "Upload complete"
+                )
+            )
+        );
+        MatcherAssert.assertThat(
+            exec("conda", "install", "--verbose", "-y", "snappy"),
```
Comment (genryxy) [inline src/test/java/com/artipie/conda/CondaSliceAuthITCase.java:170]: it is necessary to add reason of failure

File: src/test/java/com/artipie/conda/CondaSliceAuthITCase.java [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.testcontainers.Testcontainers;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.GenericContainer;
+
+/**
+ * Conda adapter integration test.
+ * @since 0.3
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```
Comment (genryxy) [inline src/test/java/com/artipie/conda/CondaSliceAuthITCase.java:36]: `since` tag is incorrect

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, two minor comments

**Comment by olenagerasimova** [comment]
@genryxy thanks, coorected

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Pull request merged**